securedrop_client/queue.py

```diff
@@ -103,7 +103,12 @@ def re_add_job(self, job: ApiJob) -> None:
         '''
         job.remaining_attempts = DEFAULT_NUM_ATTEMPTS
         priority = self.JOB_PRIORITIES[type(job)]
-        self.queue.put_nowait((priority, job))
+        try:
```

Comment:
this is out of scope and can be made into a separate pr, right?  or was this needed for optimizing UI during a sync? also see how we are moving towards a continuous background sync that only ever runs one sync at a time: https://github.com/freedomofpress/securedrop-client/pull/739